src/lib/spinel/spinel.h

```diff
@@ -1568,7 +1568,15 @@ enum
     SPINEL_PROP_PHY_PCAP_ENABLED   = SPINEL_PROP_PHY__BEGIN + 8,  ///< [b]
     SPINEL_PROP_PHY_CHAN_PREFERRED = SPINEL_PROP_PHY__BEGIN + 9,  ///< [A(C)]
     SPINEL_PROP_PHY_FEM_LNA_GAIN   = SPINEL_PROP_PHY__BEGIN + 10, ///< dBm [c]
-    SPINEL_PROP_PHY__END           = 0x30,
+
+    /// Signal the max power for a channel
+    /** Format: `Cc`
+     *
+     * First byte is the channel then the max transmit power, write-only.
+     */
+    SPINEL_PROP_PHY_CHAN_MAX_POWER = SPINEL_PROP_PHY__BEGIN + 11,
```

Comment:
Can you please add the new property in `spinel.c` in `spinel_prop_key_to_cstr()` switch statement? Thanks.